packages/node/test/unit/dev.test.ts

```diff
@@ -24,6 +26,34 @@ function testForkDevServer(entrypoint: string) {
   });
 }
 
+(nodeMajor >= 18 ? test : test.skip)(
+  'runs an edge function that uses `WebSocket`',
+  async () => {
+    const child = testForkDevServer('./edge-websocket.js');
+    try {
+      const result = await readMessage(child);
+      if (result.state !== 'message') {
+        throw new Error('Exited. error: ' + JSON.stringify(result.value));
+      }
+
+      const { address, port } = result.value;
+      const response = await fetch(
+        `http://${address}:${port}/api/edge-websocket`
```

Comment:
there is a bug in node-fetch that requires to don't use localhost
https://github.com/node-fetch/node-fetch/issues/1624

Reply:
Ha, their recommendation is don't use node-fetch if you're using Node 18.